cli/cmd/bb/bb.go

```diff
@@ -2,29 +2,56 @@ package main
 
 import (
 	"flag"
+	"os"
 
 	"github.com/buildbuddy-io/buildbuddy/cli/arg"
 	"github.com/buildbuddy-io/buildbuddy/cli/bazelisk"
+	"github.com/buildbuddy-io/buildbuddy/cli/log"
 	"github.com/buildbuddy-io/buildbuddy/cli/login"
 	"github.com/buildbuddy-io/buildbuddy/cli/parser"
+	"github.com/buildbuddy-io/buildbuddy/cli/plugin"
 	"github.com/buildbuddy-io/buildbuddy/cli/remotebazel"
 	"github.com/buildbuddy-io/buildbuddy/cli/sidecar"
+	"github.com/buildbuddy-io/buildbuddy/cli/terminal"
 	"github.com/buildbuddy-io/buildbuddy/cli/tooltag"
 	"github.com/buildbuddy-io/buildbuddy/cli/version"
 )
 
 func main() {
 	flag.Parse()
+	exitCode, err := run()
+	if err != nil {
+		log.Fatal(err)
+	}
+	os.Exit(exitCode)
+}
+
+func run() (exitCode int, err error) {
+	// Load plugins
+	plugins, err := plugin.LoadAll()
+	if err != nil {
+		return 0, err
```

Comment:
nit: i think it doesn't matter right now (because we don't hit exit) - but i think we probably want a non-zero exit code in this case (and the errors below)

Reply:
made it `-1` instead.

(I didn't make a const for it, because I thought the made the code a bit unconventional -- it looks too much at a glance like we are returning something that is intended to be used when we actually want it to be discarded.)